src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -41,6 +41,29 @@
 _endpoint_type_map = {"chat": "v1/chat/completions", "completions": "v1/completions"}
 
 
+def _check_model_args(
+    parser: argparse.ArgumentParser, args: argparse.Namespace
+) -> argparse.Namespace:
+    """
+    Check if model name is provided.
+    """
+    if not args.subcommand and not args.model:
+        parser.error("The -m/--model option is required and cannot be empty.")
+    return args
+
+
+def _check_compare_args(
+    parser: argparse.ArgumentParser, args: argparse.Namespace
+) -> argparse.Namespace:
+    """
+    Check compare subcommand args
+    """
+    if args.subcommand == "compare":
+        if not args.config and not args.files:
+            parser.error("Either --config or --files option must be specified.")
```

Comment:
```suggestion
            parser.error("Either the --config or --files option must be specified when comparing.")
```

Reply:
Updated. The error format already specifies that we are in `compare` so I don't think we need to highlight "when comparing" again.

Example:
```
$ genai-perf compare
usage: genai-perf compare [-h] [--config CONFIG | -f FILES [FILES ...]]
genai-perf compare: error: Either the --config or --files option must be specified.
```

Reply:
I agree with this approach, though it looks like "the" was not pushed.